### ISSUE TITLE
Alive.txt contains no data which further fails WAF, Screenshots and vulnerability scans

### ISSUE THREAD

**Issue by hax0rgb** [yogeshojha/rengine#739]
### Platform Details

**ReNgine version**: 1.3.3
**OS**: Ubuntu 22.10 x64 (VPS)

### Issue Summary

I have tested 3 targets and in all those cases `alive.txt` contains no data. Therefore, tools like `Wafwoof`, `Eyewitness` and `nuclei` do not output any data too.

### Steps to Reproduce

1. Run `Full Scan` against any target like `testphp.vulnweb.com`.
2. Observe `alive.txt` contains no output.

<img width="595" alt="Screenshot 2022-11-06 at 11 58 48 PM" src="https://user-images.githubusercontent.com/13043764/200188532-14003af9-59df-4719-ad53-c63178aa269c.png">

**Comment by roisec** [comment]
I think this is the same issue 
#738

**Comment by kr0no** [comment]
Yup, same issue as #738. Just made the PR #740. Feel free to test it, should be fixed.

**Comment by yogeshojha** [comment]
Thank you for reporting this, there was an issue with httpx as they changed the output json format. 

This is now fixed in 1.3.4
